pkg/helpers/values.go

```diff
@@ -0,0 +1,27 @@
+package helpers
+
+import "os"
+
+const (
+	// The top most value here will be the default as ints default to 0
```

Comment:
I don't think that's the case, there's no int here?

Reply:
Right, that was when I implemented it with a go Enum. Can be removed.